exocom-server/src/cli-logger.ls

```diff
@@ -0,0 +1,60 @@
+require! {
+  'chalk' : {cyan, dim, green, red, magenta, yellow}
+}
+
+
+# Logs events in a human-friedly format to the CLI
+class CliLogger
+
+  log: (text) ~>
+    console.log text
+
+
+  write: (text) ~>
+    process.stdout.write text
+
+
+  log-header: (text) ->
+    @log dim text
+
+
+  log-websockets-online: (port) ~>
+    @log "ExoCom WebSocket listener online at port #{cyan port}"
```

Comment:
Breaking change: This no longer prints the Exocom version. We print it at the beginning, so it was not needed here and just made waiting for this output unnecessarily difficult.